pkgs/development/compilers/graalvm/default.nix

```diff
@@ -0,0 +1,13 @@
+{ callPackage }:
+
+let
+  cePackages = callPackage ./community-edition { };
+in
+cePackages
+// rec {
+  buildGraalvm = callPackage ./buildGraalvm.nix;
+
+  graalvm-oracle_22 = callPackage ./graalvm-oracle { version = "22"; };
+  graalvm-oracle_17 = callPackage ./graalvm-oracle { version = "17"; };
+  graalvm-oracle = graalvm-oracle_22;
+}
```

Comment:
I think the community edition should be the default here and the Oracle should be the other.

But actually maybe this separation doesn't make sense at all. We can just put all packages in one file I think.

Reply:
There isn't a default, they're both in `graalvmPackages`, with extra aliases outside. It's `graalvm-ce` (same as before) and `graalvm-oracle`.

I will merge  `community-edition/default.nix` into this one, and we can have just one scope for everything